pkgs/tools/security/pass/rofi-pass.nix

```diff
@@ -1,15 +1,14 @@
-{ stdenv, fetchgit
+{ stdenv, fetchurl
 , pass, rofi, coreutils, utillinux, xdotool, gnugrep, pwgen, findutils, gawk
 , makeWrapper }:
 
 stdenv.mkDerivation rec {
   name = "rofi-pass-${version}";
-  version = "1.3.2";
+  version = "1.4.3";
 
-  src = fetchgit {
-    url = "https://github.com/carnager/rofi-pass";
-    rev = "refs/tags/${version}";
-    sha256 = "1xqp8s0yyjs2ca9mf8lbz8viwl9xzxf5kk1v68v9hqdgxj26wgls";
+  src = fetchurl {
```

Comment:
I agree. Using more specific fetchers also allows us to implement tools that know how to check for the newest release on for example Github. Using URLs or plain `fetchgit` would need guessing from URIs.